Index: client/src/androidTest/java/com/jraska/github/client/http/ReplayHttpComponent.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- client/src/androidTest/java/com/jraska/github/client/http/ReplayHttpComponent.kt	(revision Local Version)
+++ client/src/androidTest/java/com/jraska/github/client/http/ReplayHttpComponent.kt	(revision Shelved Version)
@@ -50,7 +50,7 @@
 
       val configuration = OkReplayConfig.Builder()
         .tapeRoot(AndroidTapeRoot(InstrumentationRegistry.getContext(), testClass))
-        .defaultMode(TapeMode.READ_ONLY)
+        .defaultMode(TapeMode.READ_WRITE)
         .sslEnabled(true)
         .interceptor(REPLAY_INTERCEPTOR)
         .build();
@@ -72,7 +72,7 @@
       return OkHttpClient.Builder()
         .addInterceptor(HttpLoggingInterceptor().setLevel(HttpLoggingInterceptor.Level.BASIC))
         .addInterceptor(REPLAY_INTERCEPTOR)
-        .addNetworkInterceptor() { _ -> throw UnsupportedOperationException(NETWORK_ERROR_MESSAGE) }
+//        .addNetworkInterceptor() { _ -> throw UnsupportedOperationException(NETWORK_ERROR_MESSAGE) }
         .build()
     }
 
